x-pack/legacy/plugins/siem/public/components/timeline/helpers.tsx

```diff
@@ -71,18 +71,33 @@ const buildQueryForAndProvider = (
     }, '')
     .trim();
 
-export const buildGlobalQuery = (dataProviders: DataProvider[], browserFields: BrowserFields) =>
+export const buildGlobalQuery1 = (dataProviders: DataProvider[], browserFields: BrowserFields) =>
```

Comment:
`buildGlobalQuery1` doesn't appear to have any usages -- is this leftover from testing?

Reply:
sure is thanks